alpha/declcfg/load.go

```diff
@@ -168,3 +168,20 @@ func readYAMLOrJSON(r io.Reader) (*DeclarativeConfig, error) {
 	}
 	return cfg, nil
 }
+
+func LoadFile(root fs.FS, file fs.File, path string) (*DeclarativeConfig, error) {
```

Comment:
```suggestion
func LoadFile(root fs.FS, file io.Reader, path string) (*DeclarativeConfig, error) {
```

Does this work?

Reply:
Works.  Tested for both regular and veneer rendering.